components/occurrence/List.vue

```diff
@@ -63,11 +63,12 @@
 </template>
 
 <script setup lang="ts">
-import { EntityOccurrence } from '../../utils/entities/occurrence'
+import {useOccurrenceSorting} from "~/utils/score";
```

Comment:
An import for utils is not required. If your local typescript setup is complaining keep the nuxt server running while developing so it can generate the types on the fly.